prime-dotnet-webapi/Services/EmailService.cs

```diff
@@ -246,6 +246,39 @@ public async Task SendEnrolleeRenewalEmails()
             }
         }
 
+
+        public async Task SendEnrolleeUnsignedToaReminderEmails()
+        {
+            var enrollees = await _context.Enrollees
+                .Include(e => e.EnrolmentStatuses)
```

Comment:
Generally, you don't need to do `.Include` and `.Select` in the same query. `.Select` does a projection, which tells EF to join the tables it needs to get the data. It will automatically join to the `EnrolmentStatus` table to compute `ApprovedDate`, so this `.Include`is ignored.